pkg/products/amqonline/addressspaceplans.go

```diff
@@ -118,9 +118,9 @@ func GetDefaultAddressSpacePlans(ns string) []*v1beta2.AddressSpacePlan {
 				LongDescription:  "Messaging infrastructure based on Apache Qpid Dispatch Router and Apache ActiveMQ Artemis and MQTT support. This plan allows an unlimited number of routers (minimum 2) and brokers, and is suitable for applications where you do not want to impose any restrictions.",
 				AddressSpaceType: "standard",
 				ResourceLimits: v1beta2.AddressSpacePlanResourceLimits{
-					Broker:    "10000.0",
-					Router:    "10000.0",
-					Aggregate: "10000.0",
+					Broker:    10000.0,
+					Router:    1000.0,
```

Comment:
I notice this changed from 10000 to 1000 any concern with that? I assume we just copied this in

Reply:
Good spotting @maleck13 . Will create a PR to fix this now

Reply:
PR up now - https://github.com/integr8ly/integreatly-operator/pull/539